packages/common/mui/src/lib/interactive-area-chart/x-axis-date-tick/x-axis-date-tick.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+import { useTheme } from '@mui/material';
+import { useIntl } from 'react-intl';
+
+const SelectedDate = (x, y) => {
+  const theme = useTheme();
+  return (
+    <svg
+      width="88"
+      height="48"
+      viewBox="0 0 88 48"
+      x={x}
+      y={y}
+      xmlns="http://www.w3.org/2000/svg"
+      fill={theme.palette.background.paper}
+    >
+      <g filter="url(#filter0_d)" style={{ borderRadius: '16px' }}>
+        <rect
+          x="6"
+          y="6"
+          width="75"
+          height="35"
+          fill="#EAFCFD"
+          stroke="#13BBC2"
+          rx="6"
+        />
+        <rect x="6" y="6" width="75" height="35" stroke="#13BBC2" rx="6" />
```

Comment:
Maybe grab the theme colors in here, this is pretty ugly hah

Reply:
I'll add the colors in. If you would like me to spend more time to try to come up with a cleaner version lmk